en/Editing and formatting/Properties.md

```diff
@@ -168,7 +194,7 @@ Obsidian comes with a set of default properties:
 
 ### Deprecated properties
 
-These properties are deprecated in 1.4. Please do not use them any more:
+These properties are deprecated in 1.4. Please do not use them anymore:
```

Comment:
Hmm. I think the change may be technically correct - but to my English-speaking eye, the `anymore` looks like a typo - I don't remember ever seeing it written as one word,

https://www.scribbr.com/commonly-confused-words/anymore-vs-any-more/

Reply:
Huh, I didn't know there was that big a difference between those two. In American English, _anymore_ is technically correct, while in UK English _any more_ is always used.

I guess the real question then is "Do the help docs conform to American English or UK English?"

Reply:
The help docs [Style Guide](https://help.obsidian.md/Contributing+to+Obsidian/Style+guide) say to follow the Google developer style guide, and the Google style guide says to prefer American spelling. 

Thus, I think _anymore_ is correct.

Reply:
Technically both can work, but `any more` sounds better to my American ears as while there is a `Please` in the sentence, it's more of an imperative. 

`He does not work here anymore`
versus
`Do we have any more cheesecake?`


Cambridge has a good comparison on the differences: https://dictionary.cambridge.org/grammar/british-grammar/any-more-or-anymore